src/components/Map/USACountyMap.tsx

```diff
@@ -41,25 +41,25 @@ interface USACountyMapProps {
   showCounties: boolean;
 }
 
+const projection = geoAlbersUsaTerritories().scale(1070).translate([400, 300]);
+
 const USACountyMap = React.memo(
   ({ stateClickHandler, showCounties }: USACountyMapProps) => {
     const [tooltipContent, setTooltipContent] = useState('');
     const locationSummaries = useSummaries();
 
-    const getFillColor = (fips: string) => {
-      const summary = (locationSummaries && locationSummaries[fips]) || null;
-      return colorFromLocationSummary(summary);
-    };
-
-    const projection = geoAlbersUsaTerritories()
-      .scale(1070)
-      .translate([400, 300]);
+    const getFillColor = useCallback(
+      (fips: string) => {
+        const summary = (locationSummaries && locationSummaries[fips]) || null;
+        return colorFromLocationSummary(summary);
+      },
+      [locationSummaries],
+    );
 
     const onMouseLeave = () => setTooltipContent('');
 
-    return (
-      <USMapWrapper>
-        {/** Map with shaded background colors for states. */}
+    const mapContent = useMemo(
```

Comment:
Split out the bulk of the map so that it doesn't re-render whenever the tooltip changes (fixes Chris's complaint about slow renders when you move your mouse around on the map).